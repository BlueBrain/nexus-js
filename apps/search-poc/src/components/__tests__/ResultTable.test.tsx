import * as renderer from "react-test-renderer";
import ResultsTable from '../ResultTable.tsx';
import React from "react";
import { shallow } from "enzyme";


it("ResultsTable renders correctly with header properties", () => {
  const props = { headerProperties: [], items: [{id: 'id'}], total: 1, onRowClick: jest.fn};
  
  const result = shallow(<ResultsTable {...props} />);
  expect(result).toMatchSnapshot();
});