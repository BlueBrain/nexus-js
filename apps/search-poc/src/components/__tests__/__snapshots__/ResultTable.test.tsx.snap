// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsTable renders correctly with header properties 1`] = `
<div
  className="result-table"
>
  <Table
    bordered={true}
    childrenColumnName="children"
    className=""
    columns={Array []}
    dataSource={
      Array [
        Object {
          "id": "id",
        },
      ]
    }
    indentSize={20}
    loading={false}
    locale={Object {}}
    onRow={[Function]}
    pagination={false}
    rowKey="key"
    showHeader={true}
    size="default"
    sortDirections={
      Array [
        "ascend",
        "descend",
      ]
    }
    title={[Function]}
    useFixedHeader={false}
  />
</div>
`;
